### PR TITLE
asset management

### PR DESCRIPTION
**Pull request by sureshhemal**

### CODE REVIEW

File: resources/js/Components/Frontend/Homepage/About.vue [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 <template>
     <div class="max-w-7xl mx-auto py-16 px-4 sm:px-6 sm:py-12 sm:px-8 sm:grid sm:grid-cols-3 sm:gap-x-8">
         <div class="pl-2 text-left sm:text-center">
-            <Heading>Who we are</Heading>
+            <Heading>{{ data.title }}</Heading>
         </div>
         <div class="flex items-center mt-12 sm:mt-0 sm:col-span-2 align-middle">
-            <Content>{{about}} <Anchor>...Read More >>></Anchor></Content>
+            <Content>{{ data.description }}
+                <Anchor>...Read More >>></Anchor>
```
Comment (Kushan1994) [inline resources/js/Components/Frontend/Homepage/About.vue:21]: eka on nee

File: commits.txt [hunk 2]
```diff
@@ -0,0 +1,307 @@
+commit 67d4fc6886212691e9a649cace5276b1a170e564
```
Comment (Kushan1994) [inline commits.txt:1]: meka mokakd

**Comment by Kushan1994** [comment]
sonar fail

File: resources/js/Components/Frontend/Homepage/About.vue (hunk 1 above)
Comment (sureshhemal) [reply resources/js/Components/Frontend/Homepage/About.vue:21]: its not my component.

File: commits.txt (hunk 2 above)
Comment (sureshhemal) [reply commits.txt]: resolved

**Comment by sureshhemal** [comment]
> sonar fail

image
Sonar cloud identifies those as duplications. But these are not as I know. Lets neglect it

**Review by Kushan1994 — APPROVED** [review]

**Pull request merged**